### PR TITLE
[READY] Adds two clown-spection scanners that print clowncrypted reports

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Adds a fake version of the N-spect scanner. It looks and examines exactly like the original but is actually a device of mass pranking.
It plays the old "WOODY" sound when printing reports, there are a few sound modes that can be cycled through with a multitool (normal N-spect scanner, woody, honking, bababooey, bababooey (varied) ).
It starts with the normal 5 seconds printing delay, but can be screwdrivered to make it print with only a 1 second delay. This can be undone by screwdrivering.
The reports it prints are useless garbage.

The fake scanner can be obtained from the black market uplink.

This also adds the bananium Clown-spect scanner, a more "powerful" but non-stealthy version of the fake N-spect scanner.
clowninspector
Made from a fake N-spect scanner, three sticky tapes and five bananium sheets. It has one more print time setting - HONK (0.1 second). Reports printed at HONK will disintegrate into water when alt-clicked. It also has one more sound setting, bwoink. Unlike the fake N-spect scanner, the bananium Clown-spect scanner has it's own examine, sprite, and examine_more.

Some minor changes to N-spect scanner code that do not change how it functions in any way were also made.

bababooey sound effects taken from the bababooey PR, angryboat.ogg is from [here](https://freesound.org/people/funnyman850/sounds/194812/)

## Why It's Good For The Game
Fun and a reason to use the black market uplink.

## Changelog
:cl:
add: You can now buy fake N-spect scanners using the black market uplink.
add: You can now upgrade that fake N-spect scanner to a bananium clown-spect scanner by crafting it with three sticky tapes and five bananium sheets.
/:cl:

### CODE REVIEW

File: code/game/objects/items/documents.dm [hunk 1]
```diff
@@ -115,3 +128,90 @@
...
+			to_chat(user, "<span class='notice'>You set the device's bleep setting to honk mode")
+		else
+			print_sound = 'sound/items/biddledeep.ogg'
+			to_chat(user, "<span class='notice'>You set the device's bleep setting to classic mode")
+	else if(I.tool_behaviour == TOOL_SCREWDRIVER)
+		if(print_time == 0)
+			print_time = 5 SECONDS
+			to_chat(user, "<span class='notice'>You set the device's scanning speed to SLOW.")
+		else
+			print_time = 0
```
Comment (coiax) [inline code/game/objects/items/documents.dm:171]: A 0 print time means they can just spam the floors with shit that no one cares about.

I realise this is your intent, and I strongly disagree.

Comment (interestingusernam3) [reply code/game/objects/items/documents.dm:171]: New janitor content

**Comment by RaveRadbury** [comment]
I hated the experience of the period of time that you're referring to, but seeing this PR did make me laugh. Could you make the reports delete after a short period of time? That might give it a better shot of going through.

File: strings/clown_nonsense.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
...you down Never gonna run around and desert you Never gonna make you cry Never gonna say goodbye Never gonna tell a lie and hurt you  Never gonna give you up Never gonna let you down Never gonna run around and desert you Never gonna make you cry Never gonna say goodbye Never gonna tell a lie and hurt you  (Ooh, give you up) (Ooh, give you up) Never gonna give, never gonna give (Give you up) Never gonna give, never gonna give (Give you up)  We've known each other for so long Your heart's been aching, but You're too shy to say it Inside, we both know what's been going on We know the game and we're gonna play it  I just wanna tell you how I'm feeling Gotta make you understand  Never gonna give you up Never gonna let you down Never gonna run around and desert you Never gonna make you cry Never gonna say goodbye Never gonna tell a lie and hurt you  Never gonna give you up Never gonna let you down Never gonna run around and desert you Never gonna make you cry Never gonna say goodbye Never gonna tell a lie and hurt you  Never gonna give you up Never gonna let you down Never gonna run around and desert you Never gonna make you cry Never gonna say goodbye Never gonna tell a lie and hurt you"
```
Comment (MrDoomBringer) [inline strings/clown_nonsense.json:14]: hmm good

File: code/game/objects/items/documents.dm (hunk 1 above)
Comment (MrDoomBringer) [reply code/game/objects/items/documents.dm:171]: clowns can already spam floors with bananas, which is arguably way worse than this
make this cost bananium or something as a result if it doesn't already

File: code/game/objects/items/documents.dm [hunk 3]
```diff
@@ -115,3 +128,90 @@
...
+			to_chat(user, "<span class='notice'>You set the device's scanning speed setting to LIGHTING FAST.")
```
Comment (ATH1909) [inline code/game/objects/items/documents.dm:172]:
```suggestion
			to_chat(user, "<span class='notice'>You set the device's scanning speed setting to LIGHTNING FAST.")
```

File: strings/clown_nonsense.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+	"HONK",
+	"HONK!",
+	"HONK!!!",
+	"HENK",
+	"HOOOOOOONK"
+    ],
+
+    "rare": [
+	"HOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOONK!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!",
+	"\n░░░░░▄▄▄▄▀▀▀▀▀▀▀▀▄▄▄▄▄▄░░░░░░░\n░░░░░█░░░░▒▒▒▒▒▒▒▒▒▒▒▒░░▀▀▄░░░░\n░░░░█░░░▒▒▒▒▒▒░░░░░░░░▒▒▒░░█░░░\n░░░█░░░░░░▄██▀▄▄░░░░░▄▄▄░░░░█░░\n░▄▀▒▄▄▄▒░█▀▀▀▀▄▄█░░░██▄▄█░░░░█░\n█░▒█▒▄░▀▄▄▄▀░░░░░░░░█░░░▒▒▒▒▒░█\n█░▒█░█▀▄▄░░░░░█▀░░░░▀▄░░▄▀▀▀▄▒█\n░█░▀▄░█▄░█▀▄▄░▀░▀▀░▄▄▀░░░░█░░█░\n░░█░░░▀▄▀█▄▄░█▀▀▀▄▄▄▄▀▀█▀██░█░░\n░░░█░░░░██░░▀█▄▄▄█▄▄█▄████░█░░░\n░░░░█░░░░▀▀▄░█░░░█░█▀██████░█░░\n░░░░░▀▄░░░░░▀▀▄▄▄█▄█▄█▄█▄▀░░█░░\n░░░░░░░▀▄▄░▒▒▒▒░░░░░░░░░░▒░░░█░\n░░░░░░░░░░▀▀▄▄░▒▒▒▒▒▒▒▒▒▒░░░░█░\n░░░░░░░░░░░░░░▀▄▄▄▄▄░░░░░░░░█░░\n",
```
Comment (ATH1909) [inline strings/clown_nonsense.json:13]: nice

File: strings/clown_nonsense.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+	"amogus",
+	"based",
+	"cringe",
+	"kek",
+	"trolled",
+	"derp",
+	"gaming",
+	"BASED",
+	"gamer",
+	"fart",
```
Comment (ATH1909) [inline strings/clown_nonsense.json:28]:
```suggestion
	"fart",
	"bruh moment",
	"redpilled",
	"bluepilled",
	"kawaii",
```

**Comment by ATH1909** [comment]
> I hated the experience of the period of time that you're referring to, but seeing this PR did make me laugh. Could you make the reports delete after a short period of time? That might give it a better shot of going through.
> 
> Even the infinite cream pie canon has a pie regen rate, some form of limitation should exist.

Or, like, make it delete old reports to make new ones when it reaches, like, 20 reports or whatever.

**Comment by Farquaar** [comment]
What does an N-Spect scanner even do?

File: code/game/objects/items/documents.dm (hunk 1 above)
Comment (TheVekter) [reply code/game/objects/items/documents.dm:171]: Nah, we really need to have at least a short delay on this. We had another item that people could spam paper with and it caused a fair bit of lag.

File: strings/clown_nonsense.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+	"bruh moment",
+	"redpilled",
+	"bluepilled",
+	"kawaii",
+	"poggers",
+	"pogchamp"
```
Comment (TheVekter) [inline strings/clown_nonsense.json:34]: Pepega
sadge
gachiGASM

**Comment by interestingusernam3** [comment]
How the FUCK does this break regal rats?

File: code/game/objects/items/documents.dm (hunk 1 above)
Comment (interestingusernam3) [reply code/game/objects/items/documents.dm]: Changed.

**Review by Twaticus — APPROVED** [review]

**Comment by interestingusernam3** [comment]
https://user-images.githubusercontent.com/51925758/119031067-f8855a80-b9aa-11eb-8d74-f3fd4e69e5b9.mov

**Comment by interestingusernam3** [comment]
FUCK

**Comment by interestingusernam3** [comment]
I don't know why this adds trapdoor icons.

**Pull request closed without merging**